riscv_isa.cpp

```diff
@@ -1,489 +1,810 @@
+/*
+*
+* @file           riscv_isa.cpp
+* @version        1.0
+* 
+*
+* @date           21 June 2016
+* @brief          The ArchC RISC-V functional model
+*
```

Comment:
Would you please add a comment here, if possible, to the link of the RISC-V specification you used to write this model. I suppose it is the riscv-spec-v2.0.pdf file, right?